Client.php

```diff
@@ -402,8 +412,9 @@ public function setCloseOnDestruct($flag)
     }
     protected function convertHost()
     {
-        if (preg_match('#^(tcp|unix)://(.*)$#', $this->host, $matches)) {
+        if (preg_match('#^(tcp|tls|unix)://(.*)$#', $this->host, $matches)) {
             if($matches[1] == 'tcp') {
```

Comment:
Missing a check against `$matches[1] == 'tls'`, so TLS is hitting the unix socket code path.

Reply:
Yes, I forget if ($matches[1] == 'tcp' || $matches[1] == 'tls') {